client/components/timezone-dropdown/index.jsx

```diff
@@ -3,50 +3,79 @@
  * External Dependencies
  */
 import React from 'react';
-import noop from 'lodash/noop';
 
 /**
  * Internal dependencies
  */
+import wpcom from 'lib/wp';
 import Dropdown from 'components/select-dropdown';
-import i18n from 'lib/mixins/i18n';
 
-export default React.createClass( {
-	displayName: 'TimezoneDropdown',
+/**
+ * Module variables
+ */
+const { Component, PropTypes } = React;
+const noop = () => {};
+const undocumented = wpcom.undocumented();
+
+class TimezoneDropdown extends Component {
+	constructor() {
+		super();
 
-	propTypes: {
```

Comment:
Do we have a preference for this pattern? I also hadn't seen it before in our codebase, but maybe I'm just missing it.